Earl-Grey-Demo/Earl-Grey-Demo/SecondViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Alex on 21/02/2017.
+// Copyright (c) 2017 Alex Curran. All rights reserved.
+//
```

Comment:
yup, also added a licence to the wrapper API so others can use it https://github.com/novoda/ios-demos/pull/4/commits/fe3327e86467ad661b298e3b7694a2659cd31cf6